src/lib/layout/api-endpoint.ts

```diff
@@ -1,3 +1,5 @@
+import path from 'node:path'
```

Comment:
So, this is clever but it will break on windows. You actually want
```suggestion
import path from 'node:path/posix'
```